index.js

```diff
@@ -29,4 +41,7 @@ class versionRouter {
   }
 }
 
-module.exports = versionRouter
+module.exports = {
+  versionRouter,
+  RouteVersionUnmatchedError
```

Comment:
let's revert so that we do not change the export structure at this time.
we'd probably anyway need to bump major version due to calling next with an error if no default route is used

Reply:
Okay.  Rolled back and exposed the class as a static property of the class as a stopgap - if you think that's too yucky, LMK and I'll remove it and also remove the instanceof tests on the errors (since the typescript error restoration stuff is no longer an issue).

Reply:
That's ok, thanks! I have an idea for a further refactor of the errors